pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -1791,6 +1791,7 @@ func Test_Run_Positive_VolumeMountControllerAttachEnabledRace(t *testing.T) {
 	<-stoppedChan
 
 	finished := make(chan interface{})
+	fakePlugin.Lock()
```

Comment:
I am not quite familiar with the reconcile process, is it expected that `fakePlugin.NewAttacher()` still get invoked even after the reconciler has been stopped? This seems a bit strange to me.

Reply:
yeah, maybe it's just in execution. .from the error output. In the code,  It 's just a routine. The call chains like this from the outside to inner.
> reconciler.Run(stopChan) ->  rc.mountAttachVolumes() -> rc.operationExecutor.MountVolume(...) -> oe.pendingOperations.Run(....) -> generatedOperations.Run()（this will new a routine to run）-> o.OperationFunc() -> deviceMountableVolumePlugin.NewDeviceMounter()  ->  fakeVolumePlugin.NewAttacher().